samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1589,11 +1589,14 @@ static void createBackup(
     // Set expire time to 14 days from now.
     Timestamp expireTime = Timestamp.ofTimeMicroseconds(TimeUnit.MICROSECONDS.convert(
         System.currentTimeMillis() + TimeUnit.DAYS.toMillis(14), TimeUnit.MILLISECONDS));
+    // Sets the version time to the current time.
+    Timestamp versionTime = Timestamp.now();
```

Comment:
Considering the issues we've seen for .NET and Go, we should change this to a timestamp that we know is valid on the backend, such as `Database.getEarliestVersionTime()`.

Reply:
Makes sense, I will update the samples

Reply:
Done.